opre_ops/ops_site/fixtures/fake_data.json

```diff
@@ -1,22 +1,4 @@
 [
-{
-    "model": "auth.user", 
-    "pk": 1, 
-    "fields": {
-        "password": "pbkdf2_sha256$260000$SPT55SvTcijon2Y3dU3U5U$gAGQU7lfr+nz+j9kovR8/iGlt1TFAbwuJo7F+dEObeM=", 
-        "last_login": null, 
-        "is_superuser": true, 
-        "username": "admin", 
-        "first_name": "", 
-        "last_name": "", 
-        "email": "email@test.com", 
-        "is_staff": true, 
-        "is_active": true, 
-        "date_joined": "2021-09-13T23:29:07.694Z", 
-        "groups": [], 
-        "user_permissions": []
-    }
-},
```

Comment:
We can revert this if you want, but it seemed weird to have the admin user in the fixtures data. Plus I don't know its password so I had to create my own admin user anyway.

Reply:
Ah, sorry, I could/should have told you the password. What alternate method would you use to create at least one user following a deployment?

Reply:
```sh
docker compose run web python manage.py createsuperuser
```

That'll walk you through the process of creating a local admin account. And since you asked this totally reasonable question, I'm also going to update the README since that will be a necessary step for someone setting up the environment for the first time.